src/components/recursive_tree/recursive_tree.tsx

```diff
@@ -61,6 +61,10 @@ export type CommonTreeProps = CommonProps &
      * that contain children
      */
     showExpansionArrows?: boolean;
+    /**
+     * Used internally to differentiate child trees from the outermost tree
+     */
+    isNested?: boolean;
```

Comment:
Added this to make life easy - not sure if there's a cleaner way without this extra boolean